beacon-chain/db/kv/kv.go

```diff
@@ -21,12 +22,30 @@ import (
 var _ iface.Database = (*Store)(nil)
 
 const (
+	// ValidatorEntryCacheSize is set to 1MB since we expect ~100K validator entries (10 times the expected size)
+	ValidatorEntryCacheSize = 1 << 20
+	// NumOfValidatorEntries is set to ~32Mb so as to allow at least 100K validators entries.
+	NumOfValidatorEntries = 1 << 25
```

Comment:
Sizes of these two variables should be the other way around, similarly to how `VotesCacheSize` and `NumOfVotes` are related to each other.

Reply:
Yeah, the name is reversed and so the sizes. I corrected ti and increased the cache to accommodate 200K validators.

Reply:
Maybe I'm missing something, the current validator size is very close to 200k. Shouldn't we budge higher to something like 500k or 1M

Reply:
I think @jmozah actually meant 1M validators because that's the actual size of the cache, right? The comment should be changed to `as to allow at least 1M validators entries`

Reply:
> Maybe I'm missing something, the current validator size is very close to 200k. Shouldn't we budget higher to something like 500k or 1M

Taking the average validators size was not a good for peak. So i took the max validator size which is 200K as you said.
I have budgeted 10 times of 200k (2Million). It looks like this now Hope the confusion clears now.
```
// NumOfValidatorEntries is the size of the validator cache entries.
// we expect to hold a max of 200K validators, so setting it to 2 million (10x the capacity).
NumOfValidatorEntries = 1 << 21

// ValidatorEntryMaxCost is set to ~64Mb to allow 200K validators entries to be cached.
ValidatorEntryMaxCost = 1 << 26
```